src/main/java/io/percy/appium/lib/ScreenshotOptions.java

```diff
@@ -3,7 +3,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import io.appium.java_client.MobileElement;
+import org.openqa.selenium.WebElement;
```

Comment:
mobile element removed in v8

Reply:
Got it, what if someone is using older appium sdk and passes mobile element

Reply:
WebElement is available in v7 so it is backward compatible. If user passes  Mobile Element then it will show error in the script due to incorrect argument type.

Reply:
But that means if user was already using MobileElement so far and they update the percy sdk they will start getting error.

Reply:
Added  backward compatibility for this.